Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesStepTransitionAnimationTrait.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  AppcuesStepTransitionAnimationTrait.swift
+//  AppcuesKit
+//
+//  Created by Matt on 2022-12-06.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import UIKit
+
+internal class AppcuesStepTransitionAnimationTrait: ContainerDecoratingTrait {
+
+    static var type: String = "@appcues/step-transition-animation"
+
+    weak var metadataDelegate: TraitMetadataDelegate?
+
+    let duration: TimeInterval
+    let easing: Easing
+
+    required init?(config: DecodingExperienceConfig, level: ExperienceTraitLevel) {
+        self.duration = config["duration"] ?? 0.3
+        self.easing = config["easing"] ?? .linear
```

Comment:
I still might explore the idea of a trait config instance that gets decoded just to see what that looks like, but this is certainly simple too!